infra/base-images/base-builder/compile

```diff
@@ -17,6 +17,11 @@
 
 echo "---------------------------------------------------------------"
 
+if [ "$SANITIZER" = "dataflow" ] && [ "$FUZZING_ENGINE" != "dataflow" ]; then
```

Comment:
nit: Actually, I wonder if it is worth putting a check in helper.py so that user doesn't need to wait for image to be setup before failure happens.

Reply:
Was thinking about that and ended up deciding that to fail loudly it's better to fail for all possible scenarios, not only though the helper script.